staging/src/k8s.io/kubelet/pkg/apis/podresources/v1alpha1/api.proto

```diff
@@ -18,6 +18,15 @@ option (gogoproto.goproto_unrecognized_all) = false;
 // node resources consumed by pods and containers on the node
 service PodResourcesLister {
     rpc List(ListPodResourcesRequest) returns (ListPodResourcesResponse) {}
+    rpc GetAllocatableResources(AllocatableResourcesRequest) returns (AllocatableResourcesResponse) {}
```

Comment:
Looks like this escaped my gaze when reading the KEP (and I noticed that it's not very well documented in the kep), could you describe a bit how you expect to use this?
Thanks a lot!

Reply:
Yeah, this is only mentioned too quickly in https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2043-pod-resource-concrete-assigments#topology-aware-scheduling. Let me post a clarification to the KEP

Reply:
https://github.com/kubernetes/enhancements/pull/2115